src/main/java/core/basesyntax/Area.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface Area {
```

Comment:
The interface name 'Area' is a noun which typically represents an entity. Interfaces should be named after the functionality they represent. In this case, a better name would be 'AreaCalculatable' or 'AreaCalculator' to clearly indicate that it provides the functionality to calculate an area.